tests/agents/test_ollama_agent.py

```
@@ -14,134 +17,108 @@
         # Test with a lightweight model
-        config = {'model': 'gpt-oss:20b', 'temperature': 0.1}
+        config = {"model": "gpt-oss:20b", "temperature": 0.1}
         agent = OllamaAgent(config)
-        print(f"✓ Agent created successfully")
+        print("✓ Agent created successfully")
         print(f"  Model: {agent.model}")
         print(f"  Base URL: {agent.base_url}")
-        return agent
+        assert agent is not None
+        assert hasattr(agent, "model")
     except Exception as e:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)